corehq/pillows/mappings/xform_mapping.py

```diff
@@ -43,6 +43,10 @@
             "type": "date",
             "format": DATE_FORMATS_STRING
         },
+        "server_modified_on": {
```

Comment:
We don't but there is a management command we can run to update them: https://github.com/dimagi/commcare-hq/blob/9a7bf86aa0f81c6a52ccc24022a125f912702d9c/corehq/ex-submodules/pillowtop/management/commands/update_es_mapping.py#L11-L11